requirements-mac.txt

```diff
@@ -0,0 +1,221 @@
+pyobjc-framework-MapKit==10.3.1
+pyobjc-framework-MediaAccessibility==10.3.1
+pyobjc-framework-MediaLibrary==10.3.1
+pyobjc-framework-MediaPlayer==10.3.1
+pyobjc-framework-MediaToolbox==10.3.1
+pyobjc-framework-Metal==10.3.1
+pyobjc-framework-MetalFX==10.3.1
+pyobjc-framework-MetalKit==10.3.1
+pyobjc-framework-MetalPerformanceShaders==10.3.1
+pyobjc-framework-MetalPerformanceShadersGraph==10.3.1
+pyobjc-framework-MetricKit==10.3.1
+pyobjc-framework-MLCompute==10.3.1
+pyobjc-framework-ModelIO==10.3.1
+pyobjc-framework-MultipeerConnectivity==10.3.1
+pyobjc-framework-NaturalLanguage==10.3.1
+pyobjc-framework-NetFS==10.3.1
+pyobjc-framework-Network==10.3.1
+pyobjc-framework-NetworkExtension==10.3.1
+pyobjc-framework-NotificationCenter==10.3.1
+pyobjc-framework-OpenDirectory==10.3.1
+pyobjc-framework-OSAKit==10.3.1
+pyobjc-framework-OSLog==10.3.1
+pyobjc-framework-PassKit==10.3.1
+pyobjc-framework-PencilKit==10.3.1
+pyobjc-framework-PHASE==10.3.1
+pyobjc-framework-Photos==10.3.1
+pyobjc-framework-PhotosUI==10.3.1
+pyobjc-framework-PreferencePanes==10.3.1
+pyobjc-framework-PushKit==10.3.1
+pyobjc-framework-Quartz==10.3.1
+pyobjc-framework-QuickLookThumbnailing==10.3.1
+pyobjc-framework-ReplayKit==10.3.1
+pyobjc-framework-SafariServices==10.3.1
+pyobjc-framework-SafetyKit==10.3.1
+pyobjc-framework-SceneKit==10.3.1
+pyobjc-framework-ScreenCaptureKit==10.3.1
+pyobjc-framework-ScreenSaver==10.3.1
+pyobjc-framework-ScreenTime==10.3.1
+pyobjc-framework-ScriptingBridge==10.3.1
+pyobjc-framework-SearchKit==10.3.1
+pyobjc-framework-Security==10.3.1
+pyobjc-framework-SecurityFoundation==10.3.1
+pyobjc-framework-SecurityInterface==10.3.1
+pyobjc-framework-SensitiveContentAnalysis==10.3.1
+pyobjc-framework-ServiceManagement==10.3.1
+pyobjc-framework-SharedWithYou==10.3.1
+pyobjc-framework-SharedWithYouCore==10.3.1
+pyobjc-framework-ShazamKit==10.3.1
+pyobjc-framework-Social==10.3.1
+pyobjc-framework-SoundAnalysis==10.3.1
+pyobjc-framework-Speech==10.3.1
+pyobjc-framework-SpriteKit==10.3.1
+pyobjc-framework-StoreKit==10.3.1
+pyobjc-framework-Symbols==10.3.1
+pyobjc-framework-SyncServices==10.3.1
+pyobjc-framework-SystemConfiguration==10.3.1
+pyobjc-framework-SystemExtensions==10.3.1
+pyobjc-framework-ThreadNetwork==10.3.1
+pyobjc-framework-UniformTypeIdentifiers==10.3.1
+pyobjc-framework-UserNotifications==10.3.1
+pyobjc-framework-UserNotificationsUI==10.3.1
+pyobjc-framework-VideoSubscriberAccount==10.3.1
+pyobjc-framework-VideoToolbox==10.3.1
+pyobjc-framework-Virtualization==10.3.1
+pyobjc-framework-Vision==10.3.1
+pyobjc-framework-WebKit==10.3.1
+pyparsing==3.1.4
+PyPDF2==3.0.1
+python-dateutil==2.9.0.post0
+pyttsx3==2.91
+pytz==2024.1
+referencing==0.35.1
+requests==2.32.3
+rich==13.8.0
+rpds-py==0.20.0
+rsa==4.9
+six==1.16.0
+smmap==5.0.1
+soundfile==0.12.1
+streamlit==1.38.0
+streamlit-scrollable-textbox==0.0.3
+tenacity==8.5.0
+toml==0.10.2
+tornado==6.4.1
+tqdm==4.66.5
+typing_extensions==4.12.2
+tzdata==2024.1
+uritemplate==4.1.1
```

Comment:
Seems like an explosion of dependencies, do we know we need all of them. just checking.